### ISSUE TITLE
Incorrect relayerFee showing on connextscan

### ISSUE THREAD

**Issue by wanglonghong** [connext/monorepo#3010]
**Describe the bug**

Connextscan shows 0 relayer fee even though users paid.
https://amarok.connextscan.io/tx/0x60fe4cc99a6b67e3349062bd01528eaa61acea16d6d2438506b13b92420899ee

![image](https://user-images.githubusercontent.com/30057497/211822052-b08889d9-d499-4e18-87d5-487984586339.png)

The transfer object from subgraph is 

```
      {
        "id": "0x60fe4cc99a6b67e3349062bd01528eaa61acea16d6d2438506b13b92420899ee",
        "transferId": "0x60fe4cc99a6b67e3349062bd01528eaa61acea16d6d2438506b13b92420899ee",
        "nonce": "110",
        "status": "XCalled",
        "messageHash": "0xe23dabe9565a110c2e1c3d99f90d03dd93947c8b84a9540aef12893bb041d219",
        "bridgedAmt": "11433805",
        "relayerFee": "13712663340725",
        "originDomain": "1869640809",
        "destinationDomain": "1886350457",
        "canonicalDomain": "6648936",
        "transactionHash": "0x8dcb7930e85300fb2ec13121150b097acaa0eda473586490676654f55fdbb223",
        "gasPrice": "1000000",
        "gasLimit": "475017",
        "blockNumber": "64350197",
        "message": {
          "id": "0xe23dabe9565a110c2e1c3d99f90d03dd93947c8b84a9540aef12893bb041d219",
          "leaf": "0xe23dabe9565a110c2e1c3d99f90d03dd93947c8b84a9540aef12893bb041d219",
          "index": "110",
          "root": "0x051b2b677a387552d08ba634266fa23ae83168440b16334b45480333d519214b",
          "message": "0x6f7074690000000000000000000000008f7492de823025b4cfaab1d34c58963f2af5deda00000027706f6c7900000000000000000000000011984dc4465481512eb5b777e44061c158cf225900657468000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48030000000000000000000000000000000000000000000000000000000000ae774d60fe4cc99a6b67e3349062bd01528eaa61acea16d6d2438506b13b92420899ee",
          "transferId": "0x60fe4cc99a6b67e3349062bd01528eaa61acea16d6d2438506b13b92420899ee",
          "destinationDomain": "1886350457",
          "transactionHash": "0x8dcb7930e85300fb2ec13121150b097acaa0eda473586490676654f55fdbb223"
        }
      },
```
**To Reproduce**
2. 
3. 

**Additional context**

**Comment by pjenviri** [comment]
We get the relayer's fee data from `sdk.nxtpSdkUtils.getTransferById`, but the field that indicates the relayer fee has disappeared. Previously, there was `xcall_relayer_fee`. Now, we don't see the related data returned from the SDK. Could you suggest which field we can use on the SDK?

**Comment by wanglonghong** [comment]
> We get the relayer's fee data from `sdk.nxtpSdkUtils.getTransferById`, but the field that indicates the relayer fee has disappeared. Previously, there was `xcall_relayer_fee`. Now, we don't see the related data returned from the SDK. Could you suggest which field we can use on the SDK?

you can just use `relayer_fee` @pjenviri

**Comment by pjenviri** [comment]
I don't see the field: `relayer_fee` returned from SDK.

**Comment by preethamr** [comment]
That is right.
Will be fixed with this:
https://github.com/connext/monorepo/issues/3015

**Comment by sanchaymittal** [comment]
seems like this was fixed under #3015, closing it now.

**Comment by wanglonghong** [comment]
yes
